aiidalab_widgets_base/databases.py

```diff
@@ -156,7 +156,22 @@ def __init__(
         title: str = None,
         **kwargs,
     ) -> None:
-        from optimade_client import default_parameters, query_filter, query_provider
+        try:
+            from optimade_client import default_parameters, query_filter, query_provider
+        except ImportError:
+            super().__init__(
+                [
+                    ipw.HTML(
+                        """
+                        <div style="margin: 10px 0; padding: 10px; background-color: #ffcc00; color: black; border-left: 6px solid #ffeb3b;">
+                            <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
+                            &nbsp;This widget requires the <code>optimade-client</code> package to be installed.
```

Comment:
```suggestion
                            &nbsp;This widget requires the <code>optimade-client</code> package to be installed.<br>
```
To make the box narrower..
 (Or limit width by CSS)

Reply:
Wait... I revert the change, it looks ugly 😸 

<img width="974" alt="image" src="https://github.com/aiidalab/aiidalab-widgets-base/assets/8831685/273bc0fc-5302-462b-a692-68090fdd9675">